src/test/java/com/razorpay/StakeholderClientTest.java

```diff
@@ -200,4 +200,53 @@ private String getResponseCollection(){
                 "  \"count\": 1\n" +
                 "}";
     }
+
+    @Test
+    public void testuploadStakeholderDoc() throws RazorpayException {
+        JSONObject request = new JSONObject();
+        request.put("files","/Users/your_name/Downloads/sample_uploaded.pdf");
+        request.put("document_type","aadhar_front");
+
+        String mockedResponseJson = "{\n" +
+                "  \"entity\": \"account\",\n" +
+                "  \"individual_proof_of_address\": [\n" +
+                "    {\n" +
+                "      \"type\": \"aadhar_front\",\n" +
+                "      \"url\": \"https://rzp.io/i/bzDAbNg\"\n" +
+                "    }\n" +
+                "  ]\n" +
+                "}";
+        try {
+            mockResponseFromExternalClient(mockedResponseJson);
+            mockResponseHTTPCodeFromExternalClient(200);
+            Account document = stakeholderClient.uploadStakeholderDoc(ACCOUNT_ID, STAKEHOLDER_ID, request);
+            assertNotNull(document);
+            assertEquals(true,document.has("individual_proof_of_address"));
+        } catch (IOException e) {
+            assertTrue(false);
+        }
+    }
+
+    @Test
+    public void testfetchStakeholderDoc() throws RazorpayException {
+
+        String mockedResponseJson = "{\n" +
+                "  \"entity\": \"account\",\n" +
+                "  \"individual_proof_of_address\": [\n" +
+                "    {\n" +
+                "      \"type\": \"aadhar_front\",\n" +
+                "      \"url\": \"https://rzp.io/i/bzDAbNg\"\n" +
+                "    }\n" +
+                "  ]\n" +
+                "}";
```

Comment:
same here, please check if mockedResponseJson can be created as new JSONObject, then convert it to String using toString() method so that it can be passed to mockResponseFromExternalClient()

Reply:
done

Reply:
https://github.com/razorpay/razorpay-java/blob/530334834fab1783cb3abd7e26e87a56681d2383/src/test/java/com/razorpay/StakeholderClientTest.java#L211